security/pkg/pki/ca/ca.go

```diff
@@ -415,14 +417,37 @@ func (ca *IstioCA) sign(csrPEM []byte, subjectIDs []string, requestedLifetime ti
 }
 
 func (ca *IstioCA) signWithCertChain(csrPEM []byte, subjectIDs []string, requestedLifetime time.Duration, lifetimeCheck,
```

Comment:
nit: IMO, It might be better idea to implement and use a new function with new functionality rather than change the functionality of an older function. Perhaps the new function could be called "getSignedCertChain" which returns the string slice

Reply:
fixed